GenderPayGap.Core/Extensions/Crypto.cs

```diff
@@ -62,7 +61,10 @@ public static string GeneratePasscode(char[] charset, int passcodeLength)
             }
 
             //use the randome number to pick from the character set
-            Parallel.For(0, chars.Length, i => { chars[i] = charset[randomData[i] % charset.Length]; });
+            for (int i = 0; i < chars.Length; i++)
+            {
+                chars[i] = charset[randomData[i] % charset.Length];
+            }

```

Comment:
This loop is only run 7 or 8 times.
The body of the loop does something extremely fast (indexing into 2 arrays, modulus, changing a value in an array).
`Parallel.For` is confusing (makes it seem like it's doing something heavyweight